mongodbatlas/doc.go

```diff
@@ -36,30 +36,30 @@ can be used as a starting point.
 Authentication
 
 The mongodbatlas library does not directly handle authentication. Instead, when
-creating a new client, pass an http.Client that can handle authentication for
-you. The easiest and recommended way to do this is using the https://github.com/Sectorbob/mlab-ns2/gae/ns/digest
-library, but you can always use any other library that provides an http.Client.
-If you have a private and public API token pair, you can
-use it with the digest library using:
+creating a new client, pass an http.Client that can handle Digest Access authentication for
+you. The easiest way to do this is using the [digest](https://github.com/mongodb-forks/digest)
+library, but you can always use any other library that provides an `http.Client`.
+If you have a private and public API token pair, you can use it with the digest library using:
+```go
```

Comment:
go doc does not support markdown, you need to keep this indented so it's rendered as code